Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ParticipantsController.cs

```diff
@@ -282,5 +286,44 @@ public void CleanSubmissionsForProcessing()
                 .Where(s => s.Processed && s.Processing == false)
                 .Delete();
         }
+
+        // TODO: Remove this method after the job is registered
+        public ActionResult RegisterJobForDeletingLeftOverFilesInTemp()
+        {
+            var cron = "0 1 * * *";
+            this.backgroundJobs.AddOrUpdateRecurringJob(
+                "DeleteLeftOverFoldersInTemp",
+                () => this.DeleteLeftOverFoldersInTemp(),
+                cron);
+
+            return null;
+        }
+
+        // TODO: Remove this method after the job is registered
+        public void DeleteLeftOverFoldersInTemp()
```

Comment:
We will need this method after we register the job, so it will not be deleted. Instead you should figure out a proper place to put it in. Maybe in one of the helper classes?